docker-compose.yml

```diff
@@ -15,19 +15,38 @@ services:
       - "3000:3000"
     depends_on:
       - postgres
+      - redis
     environment:
       - DB_HOST
+      - REDIS_URL
+
+#  worker:
```

Comment:
The shared repo and `entrypoint.sh` is causing a problem here. We could probably make this `depends_on: app` but I suspect that's only masking that we're trying to use the same working directory in multiple containers.

Along those lines, I'm struggling to run tests locally with the all-docker setup.